test/oracle/helpers/callback.jsligo

```diff
@@ -25,10 +25,10 @@ const base_storage : Callback.storage = {
     meta : (Map.empty as map<string, bytes>)
 }
 
-const originate_from_file = (initial_storage : Callback.storage) => {
+export const originate_from_file = (initial_storage : Callback.storage) => {
     let oracle_path = "../../../src/contracts/jsligo/oracle/callback/main.jsligo";
     let iTres = Test.run( (x : Callback.storage) => x, initial_storage);
-    let [callback_addr, _, _] = Test.originate_from_file(oracle_path, "main", (list([]) as list<string>), iTres, 0 as mutez);
+    let [callback_addr, _, _] = Test.originate_from_file(oracle_path, iTres, 0 as mutez);
```

Comment:
Looks like this pattern will be KO soon : https://tezos-dev.slack.com/archives/GKH15NM8W/p1694776637168069

Reply:
Fixed it, checked w.r.t. latest LIGO `dev`